spec/models/setting_spec.rb

```diff
@@ -1,21 +1,23 @@
 require "rails_helper"
+require "models/shared/positive_present_integer"
+require "models/shared/present"
 
```

Comment:
Maybe it's just me but without reading further I wasn't sure what the word 'present' related to. How about using the word 'presence' to remove ambiguity?